domain-xample/xample-workflows/src/main/java/gov/va/vro/routes/xample/BieXampleRoutes.java

```diff
@@ -42,8 +60,13 @@ private void handleException(BieMessagePayload payload, Exception e) {
     try {
       String jsonBody = objectMapper.writeValueAsString(payload);
       log.error("FailedMessageEventBody: " + jsonBody);
-    } catch (Exception jsonException) {
+
+      metricLogger.submitCount(
+          IMetricLoggerService.METRIC.RESPONSE_ERROR, metricTagsSaveContentionEvent);
+    } catch (JsonProcessingException jsonException) {
       log.error("Error converting failed message to JSON", jsonException);
+    } catch (Exception exception) {
+      log.error(exception.toString());
```

Comment:
What do we do elsewhere for loggin exceptions?  The log above (line 67) is more what I'm used to: 'log.error("some description", e)' to print the stack trace for 'e'.

Reply:
addressed in [c9d6be3](https://github.com/department-of-veterans-affairs/abd-vro/pull/3267/commits/c9d6be3e9d5851dd928e78f2a82cc8ce0174a911).